modules/end-user-guide/partials/proc_building-custom-intellij-editor-images.adoc

```diff
@@ -0,0 +1,151 @@
+[id="building-custom-intellij-editor-images_{context}"]
+= Building custom IntelliJ editor images
+
+This procedure describes how to build an IntelliJ-based editor image using a custom archive and a custom image name.
+
+.Prerequisites
+
+* Define the following locations to publish the editor image and metadata:
++
+`__<IntelliJ-container-image>__`:: Location of the IntelliJ editor image in a registry accessible by {prod-short}.
++
+`__<URL_to_meta.yaml>__`:: HTTPS resource containing the editor metadata.
+
+* Define the build parameters:
++
+`--tag __<name:tag>__`::
+Tag to apply to the image after build.
++
+`--url __<downloadUrl>__`::
+Download URL pointing to the archive of the custom IntelliJ-based distribution, meeting following criteria: 
++
+** Linux platform.
+** Archive in `+tar.gz+` format.
+** With JetBrains Runtime (JBR).
+** Supported by Projector. The last version supported by Projector is 2020.3.
++
+.Searching the download URL for IntelliJ IDEA
+====
+. Navigate to link:https://www.jetbrains.com/idea/download/other.html[Download IntelliJ IDEA Other Versions]
+. Copy the download link for the archive: Version 2020.3, IntelliJ IDEA Community Edition, 2020.3.4 - Linux (`tar.gz`) with JBR.
+====
+
+* The following tools are available:
+** A Docker version supporting link:https://docs.docker.com/develop/develop-images/build_enhancements/[Docker BuildKit]. Docker 18.09 or greater.
+** `git`
+** `jq`
+** Java Development Kit (JDK) version 11
```

Comment:
@themr0c I'm adding an install link. Pls double-check that this is the correct link: https://docs.oracle.com/en/java/javase/11/install/installation-jdk-linux-platforms.html#GUID-737A84E4-2EFF-4D38-8E60-3E29D1B884B8

Reply:
The requirement is about any JDK version 11, not specifically the version by Oracle. For instance, OpenJDK should be fine.

Reply:
@max-cx @themr0c is right, and also, Oracle's JDK would not even be the preferred choice. Regardless, a link is not warranted here. We also don't know what platform the using.